src/app/pages/quickMaps/components/download/download.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="container-row">
+  <div class="element-container">
+    <i class="fas fa-file-image"></i>
+    <a [href]="chartDownloadPNG" target="_blank">
+      <button *ngIf="chartDownloadPNG" mat-raised-button color="primary">Save as .png</button>
```

Comment:
I think this can be done on the same line using [disabled]="chartDownloadPNG ? true : null".

Reply:
done